src/display/annotation_layer.js

```diff
@@ -915,7 +915,7 @@ class CheckboxWidgetAnnotationElement extends WidgetAnnotationElement {
     element.type = "checkbox";
     element.name = this.data.fieldName;
     if (value) {
-      element.setAttribute("checked", true);
+      element.setAttributes("checked", true);
```

Comment:
There's a extra "s" added at the end of the method, which probably explains why not all tests run with the test logs containing:
```
TEST-UNEXPECTED-FAIL | test failed firefox has not responded in 120s
TEST-UNEXPECTED-FAIL | test failed chrome has not responded in 120s
```